datahub-web-react/src/app/ingestV2/source/IngestedAssets.tsx

```diff
@@ -113,44 +198,101 @@ export default function IngestedAssets({ id, executionResult }: Props) {
     // The total number of assets ingested
     const total = totalEntitiesIngested ?? data?.searchAcrossEntities?.total ?? 0;
 
+    const ingestionContents = useMemo(
+        () => executionResult && getIngestionContents(executionResult),
+        [executionResult],
+    );
+    const otherIngestionContents = useMemo(
+        () => executionResult && getOtherIngestionContents(executionResult),
+        [executionResult],
+    );
+
     return (
         <>
             {error && <Message type="error" content="" />}
-            <HeaderContainer>
-                <TitleContainer>
-                    <Typography.Title level={5}>Ingested Assets</Typography.Title>
-                    {(loading && <Typography.Text type="secondary">Loading...</Typography.Text>) || (
-                        <>
-                            {(total > 0 && (
-                                <Typography.Paragraph type="secondary">
-                                    The following asset types were ingested during this run.
-                                </Typography.Paragraph>
-                            )) || <Typography.Text>No assets were ingested.</Typography.Text>}
-                        </>
+            <Heading type="h4" size="lg" weight="bold">
+                Ingested Assets
+            </Heading>
+            {(loading && (
+                <Text color="gray" colorLevel={600}>
+                    Loading...
+                </Text>
+            )) || (
+                <>
+                    {(total > 0 && (
+                        <Text color="gray" colorLevel={600}>
+                            The following asset types were ingested during this run.
+                        </Text>
+                    )) || <Text>No assets were ingested.</Text>}
+                </>
+            )}
```

Comment:
I find it simpler to have the layout like:

loading && ...
!loading && ...

rather than the || which makes it harder to read

Reply:
good point. taken care of